pom.xml

```diff
@@ -117,6 +117,16 @@
             <artifactId>commons-io</artifactId>
             <version>2.6</version>
         </dependency>
+        <dependency>
```

Comment:
Can we avoid these dependencies?

Reply:
Because java8 does not have its own benchmarking framework, these dependencies must be referenced.

Reply:
As you said, adding `<scope>test</scope>` would be better.